### ISSUE TITLE
Error during parsing of paket.lock

### ISSUE THREAD

**Issue by seanamosw** [mrward/monodevelop-paket-addin#13]
When expanding the `Paket Dependencies` menu item in the Solution explorer, no dependencies are listed.

When trying to use any operation on the `Paket Dependencies` menu item, such as `Restore` an error is reported in the Paket Console:

```
Error during parsing of 'xxx/paket.lock'.

Could not restore paket dependencies.
```

After adding / removing things from the paket.lock file to see what would get it to parse, it seems related to a framework restriction of `netcoreapp2.1`. I think the version of the Paket library included in the addon is unable to parse that.

**Comment by mrward** [comment]
What version of Paket are you using to generate the lock file?

I suspect the problem is that the addin includes its own copy of Paket and since Paket itself changes rapidly the addin gets out of date. I was thinking about removing this dependency as much as possible and just shelling out to paket, using whatever paket version is being used locally.

**Comment by seanamosw** [comment]
This lock file was created with Paket 5.176.4.

It looks like parsing netcoreapp2.1 was added in 5.147.0:
https://github.com/fsprojects/Paket/releases/tag/5.147.0

**Comment by mrward** [comment]
Updated to Paket 5.176.4 and that seems to handle netcoreapp2.1 in the paket.dependencies as a framework. 

This fix is in version 0.5 of the Paket addin which should be available from the main MonoDevelop addin server. Also [available to download](https://github.com/mrward/monodevelop-addins/blob/gh-pages/7.0/MonoDevelop.Paket_0.5.mpack)
